apps/coscrad-frontend/src/components/term-of-the-day/term-of-the-day.container.tsx

```diff
@@ -0,0 +1,46 @@
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { useLoadableTermById } from '../../store/slices/resources';
+import { TermOfTheDayPresenter } from './term-of-the-day.presenter';
+
+export const TermOfTheDayContainer = (): JSX.Element => {
+
+    //get termOfTheDay from config instead of hardwired const termArray
+    const { termOfTheDay } = useContext(ConfigurableContentContext);
+
```

Comment:
I think you should search through the `termOfTheDayConfig` and find today's ID. Then call the `useLoadableTermById` custom hook.

```ts
    const { termOfTheDayConfig } = useContext(ConfigurableContentContext);

    const { month, day } = UseDate(); // You'd need to write this custom hook `UseDate`

    const termOfTheDayId = termOfTheDayConfig[month][day]; // better yet, write a helper `findTermOfTheDay(termOfTheDayConfig,date)`.

   const loadableTermSearchResult = useLoadableTermById(termOfTheDayId);


  const Presenter = displayLoadableSearchResult(TermDetailFullViewPresenter);

  return (
    <Presenter {...loadableTermSearchResult} />
)
```

Reply:
Resolved